indico/modules/events/surveys/templates/management/survey.html

```diff
@@ -95,15 +95,22 @@ <h2>
     {% if submitted_surveys %}
         <div class="toolbars space-after">
             <div class="toolbar">
-                <button class="i-button icon-checkbox-checked arrow js-dropdown"  data-toggle="dropdown"></button>
-                <ul class="i-dropdown">
-                    <li><a href="#" id="select-all">
-                        {% trans %}All{% endtrans %}
-                    </a></li>
-                    <li><a href="#" id="select-none">
-                        {% trans %}None{% endtrans %}
-                    </a></li>
-                </ul>
+                <div class="group">
+                    <button class="i-button icon-checkbox-checked arrow js-dropdown" data-toggle="dropdown"></button>
+                    <ul class="i-dropdown">
+                        <li><a href="#" id="select-all">
+                                {% trans %}All{% endtrans %}
+                            </a></li>
```

Comment:
That indentation is really weird. Not sure how that originally (which was just slightly less weird) got there in the first place without someone complaining during a review ;)

```suggestion
                        <li><a href="#" id="select-all">{% trans %}All{% endtrans %}</a></li>
```

Just for reference, those would be decent ways to wrap this if we wanted to wrap it (not necessary here I'd say):

```html
<!-- option 1: go for that if you prefer some wrapping -->
<li>
  <a href="#" id="select-all">{% trans %}All{% endtrans %}</a>
</li>

<!-- option 2: more useful if the the `a` tag had many more attributes -->
<li>
  <a href="#" id="select-all">
    {% trans %}All{% endtrans %}
  </a>
</li>
```